<ui version="4.0" >
 <class>eduversumGui</class>
 <widget class="QWidget" name="eduversumGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Eduversum</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item rowspan="5" row="0" column="0" >
    <widget class="QTreeWidget" name="categoriesTreeWidget" >
     <property name="minimumSize" >
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>3332</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="headerHidden" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame0" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Suchen:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="searchLineEdit" />
      </item>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>168</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" >
       <widget class="QComboBox" name="comboBox" >
        <item>
         <property name="text" >
          <string>Alle Programme</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Nur installierte</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Nur nicht installierte</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="headerHidden" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QFrame" name="frame1" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QPushButton" name="execPushButton" >
        <property name="text" >
         <string>Starten</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="homepagePushButton" >
        <property name="text" >
         <string>Homepage</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="examplePushButton" >
        <property name="text" >
         <string>Beispieldateien</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4" >
       <widget class="QPushButton" name="showChangesPushButton" >
        <property name="text" >
         <string>Änderungen anzeigen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QFrame" name="frame2" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QPushButton" name="cancelChangesPushButton" >
        <property name="text" >
         <string>Abbrechen</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer name="horizontalSpacer_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="discardChangesPushButton" >
        <property name="text" >
         <string>Alle Änderungen verwerfen</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="applyChangesPushButton" >
        <property name="text" >
         <string>Änderungen ausführen</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QTextBrowser" name="descriptionTextBrowser" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
